### PR TITLE
#1705 - Remove Obsolete Form Definitions

### PR DESCRIPTION
**Pull request by katrina-aot**
verified the following were not used as per the ticket:
aestdashboard.json
approvedeclineoffering.json
approvedenydesignation.json
approveeducationprogram.json
createnote.json
declineeducationprogram.json
institutiondashboard.json
studentapplicationdetails.json
studentdashboard.json
trackstudentapplication.json

### CODE REVIEW

File: sources/packages/forms/src/form-definitions/partnerincomeappeal.json
Comment (andrewsignori-aot) [inline sources/packages/forms/src/form-definitions/partnerincomeappeal.json:1]: This file must be reverted. My bad, I forgot about these dynamic forms loaded by `studentrequstchange.json`.
Please revert also `studentdisabilityappeal.json` and `studentincomeappeal.json`.

**Review by andrewsignori-aot — COMMENTED** [review]
Nice work and thanks for doing the cleanup. Please take a look at the comment.

File: sources/packages/forms/src/form-definitions/partnerincomeappeal.json
Comment (katrina-aot) [reply sources/packages/forms/src/form-definitions/partnerincomeappeal.json:1]: I've added:
partnerincomeappeal.json
studentdisabilityappeal.json
studentincomeappeal.json
back into the folder

**Review by andrewsignori-aot — APPROVED** [review]
Thanks for adding the files back, look good 👍 
Please adjust the PR description to no longer mention the files added back in the last commit 😉

**Review by lewischen-aot — APPROVED** [review]
Nice work @katrina-aot. Thanks for adding the files back.

**Review by guru-aot — APPROVED** [review]
LGTM

**Pull request merged**